### ISSUE TITLE
v0.25.0 is broken because of `undefined method `end_pos' for nil:NilClass`

### ISSUE THREAD

**Issue by shawnzhu** [brigade/scss-lint#179]
I'm using release v0.25.0 on ruby 2.1.2p95 (2014-05-08 revision 45877) and it reports the same unexpected error on any scss file no matter there are linter errors or not:

```
SCSSLint::Linter::Indentation raised unexpected error linting file src/documents/styles/modules/_bootstrap.scss: 'undefined method `end_pos' for nil:NilClass'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/linter/indentation.rb:30:in `check_indentation'
/usr/local/lib/ruby/gems/2.1.0/gems/sass-3.3.8/lib/sass/tree/visitors/base.rb:36:in `visit'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/linter.rb:115:in `visit'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/linter.rb:125:in `block in visit_children'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/linter.rb:123:in `each'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/linter.rb:123:in `visit_children'
/usr/local/lib/ruby/gems/2.1.0/gems/sass-3.3.8/lib/sass/tree/visitors/base.rb:36:in `block in visit'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/linter/indentation.rb:9:in `visit_root'
/usr/local/lib/ruby/gems/2.1.0/gems/sass-3.3.8/lib/sass/tree/visitors/base.rb:36:in `visit'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/linter.rb:115:in `visit'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/linter.rb:18:in `run'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/runner.rb:43:in `block in find_lints'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/runner.rb:38:in `each'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/runner.rb:38:in `find_lints'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/runner.rb:22:in `block in run'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/runner.rb:21:in `each'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/runner.rb:21:in `run'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/lib/scss_lint/cli.rb:98:in `run'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/bin/scss-lint:8:in `block in <top (required)>'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/bin/scss-lint:6:in `tap'
/usr/local/lib/ruby/gems/2.1.0/gems/scss-lint-0.25.0/bin/scss-lint:6:in `<top (required)>'
/usr/local/bin/scss-lint:23:in `load'
/usr/local/bin/scss-lint:23:in `<main>'
Report this bug at https://github.com/causes/scss-lint/issues
```

I workaround this by rolling back to v0.24.0.
